backend/config/config.go

```diff
@@ -76,12 +86,16 @@ type Backend struct {
 
 	Authentication bool `json:"authentication"`
 
-	BTC  btcCoinConfig `json:"btc"`
-	TBTC btcCoinConfig `json:"tbtc"`
-	RBTC btcCoinConfig `json:"rbtc"`
-	LTC  btcCoinConfig `json:"ltc"`
-	TLTC btcCoinConfig `json:"tltc"`
-	ETH  ethCoinConfig `json:"eth"`
+	BTC    btcCoinConfig `json:"btc"`
+	TBTC   btcCoinConfig `json:"tbtc"`
+	RBTC   btcCoinConfig `json:"rbtc"`
+	LTC    btcCoinConfig `json:"ltc"`
+	TLTC   btcCoinConfig `json:"tltc"`
+	ETH    ethCoinConfig `json:"eth"`
+	GOETH  ethCoinConfig `json:"goeth"`
```

Comment:
I guess I can remove `GOETH` and `SEPETH` again, as I only had to add them for the nested explorer url which is now in `BlockExplorers`